melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/eux/EuxService.kt

```diff
@@ -0,0 +1,158 @@
+package no.nav.melosys.eessi.service.eux
+
+import mu.KotlinLogging
+import no.nav.melosys.eessi.integration.eux.rina_api.Aksjoner
+import no.nav.melosys.eessi.integration.eux.rina_api.EuxConsumer
+import no.nav.melosys.eessi.integration.eux.rina_api.EuxRinasakerConsumer
+import no.nav.melosys.eessi.integration.eux.rina_api.dto.EuxMelosysSedOppdateringDto
+import no.nav.melosys.eessi.integration.eux.rina_api.dto.Institusjon
+import no.nav.melosys.eessi.integration.eux.rina_api.dto.SedJournalstatus
+import no.nav.melosys.eessi.metrikker.BucMetrikker
+import no.nav.melosys.eessi.models.BucType
+import no.nav.melosys.eessi.models.SedVedlegg
+import no.nav.melosys.eessi.models.buc.BUC
+import no.nav.melosys.eessi.models.bucinfo.BucInfo
+import no.nav.melosys.eessi.models.exception.IntegrationException
+import no.nav.melosys.eessi.models.exception.NotFoundException
+import no.nav.melosys.eessi.models.exception.ValidationException
+import no.nav.melosys.eessi.models.sed.SED
+import no.nav.melosys.eessi.models.vedlegg.SedMedVedlegg
+import no.nav.melosys.eessi.service.sed.helpers.LandkodeMapper
+import org.springframework.context.annotation.Primary
+import org.springframework.retry.annotation.Retryable
+import org.springframework.stereotype.Service
+import org.springframework.util.StringUtils
+import java.util.*
+
+private val log = KotlinLogging.logger {}
+
+@Service
+@Primary // FIXME: Ganske sikker på at vi kan fjerne Primary her
```

Comment:
Jeg regner med vi hadde primary fordi vi hadde autowired i constructoren før. Tror det er bare å fjerne.

Reply:
Lar det uansett være så vi ikke får for mye å feilsøke i når dette testes på Q2

Reply:
Ingen klasse arver fra `EuxService` og ingen interface brukes så `@Primary` kan fjernes.

Reply:
Her kan man se hvorfor `@Primary` ble brukt: https://github.com/navikt/melosys-eessi/commit/ddf27df9aa056b84778d0c793f0e2d5215c7f51a#diff-a030c04ebb0ae00ed247d123124c7db15bfec46dbc0a22fd31e6baad3b792720

Reply:
Det så slik ut for meg også, men da er det rart dette ble lagt på?